generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/hooks/SchemaGeneratorHooks.kt

```diff
@@ -50,11 +51,24 @@ import kotlin.reflect.KType
  * that allow users to customize the schema.
  */
 interface SchemaGeneratorHooks {
+
+    /**
+     * Called before any generation of GraphQL schema.
+     *
+     * Allows users to do some preprocessing and generate custom schema builder instance.
+     */
+    fun willBuildSchema(queries: List<TopLevelObject>,
```

Comment:
`willBuildSchema` was executed AFTER we already build it, since I need to execute logic BEFORE we start building anything I standardized the hooks as `willBuildSchema -> BUILD -> didBuildSchema`

Reply:
fixed in https://github.com/ExpediaGroup/graphql-kotlin/pull/1817